package.json

```diff
@@ -94,7 +101,7 @@
     "build:reset": "git checkout upstream/master lib/marked.cjs lib/marked.umd.js lib/marked.esm.js marked.min.js",
     "build": "npm run rollup && npm run build:types && npm run build:man",
     "build:docs": "npm run build && node docs/build.js",
-    "build:types": "tsc --project tsconfig-types.json",
+    "build:types": "dts-bundle-generator --project tsconfig.json -o lib/marked.d.ts src/marked.ts && dts-bundle-generator --project tsconfig.json -o lib/marked.d.cts src/marked.ts",
```

Comment:
Will `dts-bundle-generator` fail when `tsc` would fail? Usually these bundlers need something like `tsc --noEmit` first.

Reply:
Hm, I didn’t notice that nothing else here is running `tsc` for validation. Do you want me to add that to `build:types` or another script?

Reply:
Ya I think we would want `build:types` to fail if `tsc` fails

Reply:
We do use `tsc` in test:types. I'm not sure if that is good enough

Reply:
Done 👍 (`tsc` with no arguments uses the file named `tsconfig.json` by default which is what you want—it already has `noEmit` and it will reflect exactly the errors you see in-editor)